less/mixins.less

```diff
@@ -1 +1,15 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.btn(@bgcolor:#008B8B, @width:200px, @font-color:#FFF) {
+    background-color: @bgcolor;
+    width: @width;
+    color: @font-color;
+    height: 45px;
+    border-radius: 7%;
+    font-size: 20px;
+}
+
+
+button {
```

Comment:
Talked about it in standup, but just a reminder:

The `mixins.less` file is only for declaring the mixins. Actually calling the button and adding the mixin as a style would happen in the other less files, like `home-page.less`